patches/net/minecraft/network/chat/contents/TranslatableContents.java.patch

```diff
@@ -13,4 +13,77 @@
 +        }
      }
  
+     @Override
+@@ -92,6 +_,13 @@
+         Language language = Language.getInstance();
+         if (language != this.decomposedWith) {
+             this.decomposedWith = language;
++
++            Component langComponent = language.getComponent(this.key);
++            if (langComponent != null) {
++                this.decomposedParts = ImmutableList.of(langComponent);
++                return;
++            }
++
+             String s = this.fallback != null ? language.getOrDefault(this.key, this.fallback) : language.getOrDefault(this.key);
+ 
+             try {
+@@ -170,28 +_,46 @@
+     public <T> Optional<T> visit(FormattedText.StyledContentConsumer<T> p_237521_, Style p_237522_) {
+         this.decompose();
+ 
+-        for (FormattedText formattedtext : this.decomposedParts) {
```

Comment:
If you don't indent this block, you don't need this patch.

Reply:
That sounds like it'd make the resulting patched code less readable, but sure :+1: